src/main/webapp/static/js/web3provider.js

```diff
@@ -0,0 +1,119 @@
+"use strict";
+
+// Unpkg imports
+const Web3Modal = window.Web3Modal.default;
+const WalletConnectProvider = window.WalletConnectProvider.default;
+const Fortmatic = window.Fortmatic;
+const evmChains = window.evmChains;
+
+// Web3modal instance
+let web3Modal;
+
+// Chosen wallet provider given by the dialog window
+let provider;
+
+
+/**
+ * Setup the orchestra
+ */
+function init() {
+
+  console.log("WalletConnectProvider is", WalletConnectProvider);
+  console.log("Fortmatic is", Fortmatic);
+  console.log("window.web3 is", window.web3, "window.ethereum is", window.ethereum);
+
+  // Check that the web page is run in a secure context,
+  // as otherwise MetaMask won't be available
+  if(location.protocol !== 'https:') {
+    // https://ethereum.stackexchange.com/a/62217/620
+    // TODO:
+  }
+
+  // Tell Web3modal what providers we have available.
+  // Built-in web browser provider (only one can exist as a time)
+  // like MetaMask, Brave or Opera is added automatically by Web3modal
+  const providerOptions = {
+    walletconnect: {
+      package: WalletConnectProvider,
+      options: {
+        // Mikko's test key - don't copy as your mileage may vary
+        infuraId: "8043bb2cf99347b1bfadfb233c5325c0",
```

Comment:
We probably need to add an Infura ID as an [environment variable](https://github.com/elimu-ai/webapp/blob/main/src/main/config/server-test/usr/local/jetty-distribution-9.4.31.v20200723/webapps/webapp-eng.xml) to our Jetty server configuration?

Reply:
Yes, that's right!

Reply:
Ok, added this as a task for me to handle: https://github.com/elimu-ai/webapp/pull/1570

Reply:
Please use this infura ID: `37c4b40db8fe47fb90a1e2ac4ff152eb`